backend/src/controllers/player.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { NextFunction, Request, Response } from "express";
+import { createPlayer as createPlayerService } from "../services/player.service";
+import { ErrorType } from "../util";
+
+export const createPlayer = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+): Promise<void> => {
+  try {
+    const code = req.query.gameCode;
+    const nickname = req.query.nickname;
+    if (typeof code != "string") {
+      res.status(400).send("Code not of type string");
+      return;
+    }
+    if (typeof nickname != "string") {
+      res.status(400).send("Nickname not of type string");
+      return;
+    }
+    const dbResp = await createPlayerService(code, nickname);
+    res.status(201).send(dbResp);
+    return;
+  } catch (e) {
+    console.log(e);
```

Comment:
I think this log could be removed as we shouldn't need the log the `400` responses and the error handler will log `next(e)`

Reply:
Good catch